### DISCUSSION TITLE
I can't get Ray to work in Javascript.

### DISCUSSION THREAD

**Discussion by cmoraleda** [spatie/ray#837] [Q&A]
Hi, I want to use it in a WordPress theme, the PHP version works perfect. I'm trying to load the standalone version (standalone.js) but none of the examples in the documentation have worked for me.

I have tried the Ray plugin for Wordpress but it only works with PHP.

Can you help me?

Thanks!

**Comment by patinthehat** [comment]
Hey @cmoraleda , could you provide more context for how you're trying to use node-ray?

**Reply by a-songac** [reply]
I am facing the same issue when following the [Recommended Standalone Initialization](https://github.com/permafrost-dev/node-ray?tab=readme-ov-file#recommended-standalone-initialization). I add to my web page the following snippet:
```
<script src="https://cdn.jsdelivr.net/npm/node-ray@latest/dist/standalone.min.js"></script>
<script>
    rayInit(window);
    ray('hello world');
</script>
```
However, I get the error `Uncaught ReferenceError: rayInit is not defined`. The same _not defined_ error occurs when invoking `Ray` directly.

I started tweaking around and noticed that when downgrading the version tag down to `1.20.9` The error disappeared and I was able to log to Ray. So the temporary solution is using version 1.20.9. As I am new to ray and just wanted to make it work (which took me much more time than expected), I have not further investigated the source of the problem yet.

**Reply by patinthehat** [reply]
@a-songac Thank you for reporting this bug. I'm looking into it now and will update this thread once I've resolved the issue.

**Reply by patinthehat** [reply]
@a-songac @cmoraleda, please try using the new v2.0 of node-ray. It should resolve the issues you've experienced.  Thank you 👍

**Comment by patinthehat** [comment]
Using node-ray 2.0+ should resolve this issue.
